src/app-layout/visual-refresh-toolbar/index.tsx

```diff
@@ -87,6 +83,23 @@ const AppLayoutVisualRefreshToolbar = React.forwardRef<AppLayoutProps.Ref, AppLa
       fireNonCancelableEvent(onToolsChange, { open });
     };
 
+    const [navigationOpen = false, setNavigationOpen] = useControllable(
+      controlledNavigationOpen,
+      onNavigationChange,
+      false,
+      {
+        componentName: 'AppLayout',
+        controlledProp: 'navigationOpen',
+        changeHandler: 'onNavigationChange',
+      }
+    );
```

Comment:
This is already happening in the shared code for all app layout versions: https://github.com/cloudscape-design/components/blob/54d2d7413c8fcd20d8776e15fedbc0f95ee94ee5/src/app-layout/index.tsx#L94-L99

Not needed here

Reply:
Somehow without this the focus management does not work: clicking on the navigation toggle does not focus on the close button of the open navigation, and viceversa. Any other hint?

Reply:
No hints, sadly, but double useControllable is definitely not a solution

Reply:
Solved